edx_notifications/server/web/templates/django/notifications_widget.html

```diff
@@ -0,0 +1,31 @@
+{% comment %}
+
+This is a "drop-in" Django template widget to instantiate the edx-notifications front end application
+which is a Backbone application that is defined in edx_notifications/server/web/static/js
+
+This HTML snippet assumes that jQuery been included in the page that is using this widget
+
+{% endcomment %}
+
+<div id='edx_notification_container'></div>
+<div id="edx_notification_pane"></div>
+
+<script src="{{ STATIC_URL }}edx_notifications/js/vendor/underscore.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/vendor/backbone.js"></script>
+
+<script src="{{ STATIC_URL }}edx_notifications/js/models/counter_icon_model.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/models/user_notification_model.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/models/notification_collection_model.js"></script>
+
+<script src="{{ STATIC_URL }}edx_notifications/js/views/counter_icon_view.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/views/notification_pane_view.js"></script>
+
+
+<script type="text/javascript">
+    /* start up the root Notification view, which will bootstrap the whole front-end */
+    var counter_icon = new CounterIconView({
+        el: $("#edx_notification_container"),
+        model: new CounterIconModel(),
+        pane_el: $("#edx_notification_pane")
```

Comment:
Is there a reason why the notification view has a separation between the container and the pane? I think it would be simpler if the view had a single element that contained both. This would then allow the view to be created without an ``el`` and then appended to a parent view wherever it was needed. I prefer using this approach for views in general because you can safely remove the view and it removes its own element, rather than an element from the DOM that was passed in. It should also make it a little easier to unit test as it doesn't require the two divs to be added as fixtures.